main.py

```diff
@@ -0,0 +1,47 @@
+def model_run(request):
+    """
+    Environment Variables:
+        api_key         :       Labelbox API Key
+        gcs_bucket      :       Name of a GCS Bucket to-be-generated
+        google_project  :       Name of the google project this Cloud Function is in
+    """
+    import json
+    from source_code.config import env_vars, create_gcs_key, get_lb_client, get_gcs_client
+    from source_code.etl import etl_job, upload_ndjson_data
```

Comment:
We need to be referring to this code direclty in the sub folders rather than as a requiremetns.txt import.  This will support our entire REPO push deployment.

Reply:
in gcloud CLI you can add your requirements.txt URL. So I'd add the source_code package to the requirements programatically